src/propTypes.ts

```diff
@@ -53,6 +53,26 @@ const defaults = {
         width: 0,
         height: 0
     },
+    responsiveEdges: {
+        sm: {
+            t: 0,
+            r: 0,
+            b: 0,
+            l: 0,
+        },
+        md: {
+            t: 0,
+            r: 0,
+            b: 0,
+            l: 0,
+        },
+        lg: {
+            t: 0,
+            r: 0,
+            b: 0,
+            l: 0,
+        },
+    },
```

Comment:
This nested structure may be terrible to work with in SiteDesigner. Try it, or else experiment with flattening it.